### DISCUSSION TITLE
[Azure Frontdoor] Automatyzacja purge cache

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#13] [Ogólna]
Niby prosta sprawa, ale trzeba poszukać trochę po Githubach, by znaleźć odpowiedź.

```bash
az extension add --name front-door
az network front-door purge-endpoint --resource-group [rgname] --name [frontdoorname] --content-paths "/*"

**Comment by kamil-mrzyglod** [comment]
Jakby ktoś szukał odpowiednika w Powershell:
```
Remove-AzFrontDoorContent -ResourceGroupName $ResourceGroupName -Name $FrontDoorName -ContentPath "/*"
```

Oczywiście konwencja nazewnicza na bakier. Link: https://docs.microsoft.com/en-us/powershell/module/az.frontdoor/remove-azfrontdoorcontent?view=azps-4.5.0

**Comment by JanuszNowak** [comment]
3 sposób jak zrobić to przez POST-a bezpośrednio z dokumentacji ms
https://docs.microsoft.com/en-us/rest/api/frontdoorservice/frontdoor/endpoints/purgecontent#request-body
